tests/e2e/serial/main_test.go

```diff
@@ -247,6 +247,109 @@ func TestSuiteScan(t *testing.T) {
+	scanSettingName := framework.GetObjNameFromTest(t) + "-mixproduct"
+	scanSetting := compv1alpha1.ScanSetting{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scanSettingName,
+			Namespace: f.OperatorNamespace,
+		},
+		ComplianceSuiteSettings: compv1alpha1.ComplianceSuiteSettings{
+			AutoApplyRemediations: false,
+			Schedule:              "0 1 * * *",
+			Suspend:               false,
+		},
+		Roles: []string{"master", "worker"},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSetting, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSetting)
+
+	// Bind the new ScanSetting to a Profile
+	bindingName := framework.GetObjNameFromTest(t) + "-binding"
+	scanSettingBinding := compv1alpha1.ScanSettingBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      bindingName,
+			Namespace: f.OperatorNamespace,
+		},
+		Profiles: []compv1alpha1.NamedObjectReference{
+			{
+				Name:     "ocp4-moderate",
+				Kind:     "Profile",
+				APIGroup: "compliance.openshift.io/v1alpha1",
+			},
+			{
+				Name:     "ocp4-moderate-node",
+				Kind:     "Profile",
+				APIGroup: "compliance.openshift.io/v1alpha1",
+			},
+			{
+				Name:     "rhcos4-moderate",
+				Kind:     "Profile",
+				APIGroup: "compliance.openshift.io/v1alpha1",
+			},
+		},
+		SettingsRef: &compv1alpha1.NamedObjectReference{
+			Name:     scanSetting.Name,
+			Kind:     "ScanSetting",
+			APIGroup: "compliance.openshift.io/v1alpha1",
+		},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSettingBinding, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSettingBinding)
+
+	// Wait until the scan completes
+	// after the scan is done
+	if err := f.WaitForSuiteScansStatus(f.OperatorNamespace, bindingName, compv1alpha1.PhaseDone, compv1alpha1.ResultNonCompliant); err != nil {
+		t.Fatal(err)
+	}
+
+	suite := &compv1alpha1.ComplianceSuite{}
+	key := types.NamespacedName{Name: bindingName, Namespace: f.OperatorNamespace}
+	if err := f.Client.Get(context.TODO(), key, suite); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert all the scans are there and completed
+	expectedScan := []string{"ocp4-moderate", "ocp4-moderate-node-worker", "ocp4-moderate-node-master", "rhcos4-moderate-worker", "rhcos4-moderate-master"}
+	for _, scan := range expectedScan {
+		found := false
+		for _, s := range suite.Status.ScanStatuses {
+			if s.Name == scan {
+				found = true
+				if s.Phase != compv1alpha1.PhaseDone {
+					t.Fatalf("expected scan %s to be done", scan)
+				}
+				// E2e tests have been flaky because of the file_permissions_var_log_kube_audit
+				// rule. This is because there is a bug in the API-server in old versions of OCP[1][2].
+				// For now, we'll just check that the scan is not inconsistent until we upgrade
+				// to a version that has the fix.
+				// [1]https://bugzilla.redhat.com/show_bug.cgi?id=2001442
```

Comment:
The errata linked to this was for 4.10, which means it should be fixed there, right?

Did you hit this case in your testing of this e2e test locally? If so, I wonder if something else is causing this since the fix appears to be in 4.10.

Reply:
Yes, this was fixed, I have it flipped the result for the suite

Reply:
Ok - so does this still manifest in an inconsistent result? I'm wondering if it's safe to remove the inconsistency check to make the test case simpler.